site/viamguides/src/air-quality/air-quality.md

```diff
@@ -0,0 +1,340 @@
+1. **Save** your updates, and confirm `YES` to apply OS customization settings. Confirm `YES` to erase data on the USB flash drive. You may also be prompted by your operating system to enter an administrator password. After granting permissions to the Imager, it will begin writing and then verifying the Linux installation to the USB flash drive.
+1. Remove the USB flash drive from your computer when the installation is complete.
+
+### Connect with SSH
+
+1. Place the USB flash drive into your Raspberry Pi and boot the Pi by plugging it in to an outlet. A red LED will turn on to indicate that the Pi is connected to power.
+1. Once the Pi is started, connect to it with SSH. From a command line terminal window, enter the following command. The text in <> should be replaced (including the < and > symbols themselves) with the user and hostname you configured when you set up your Pi.
+    ```bash
+    ssh <USERNAME>@<HOSTNAME>.local
+    ```
+1. If you are prompted “Are you sure you want to continue connecting?”, type “yes” and hit enter. Then, enter the password for your username. You should be greeted by a login message and a command prompt.
+    ![raspberry pi SSH login](assets/sshLogin.png)
+1. Update your Raspberry Pi to ensure all the latest packages are installed
+    ```bash
+    sudo apt update
+    sudo apt upgrade
+    ```
+
+### Enable communication protocols
+
+1. Launch the Pi configuration tool by running the following command
+    ```bash
+    sudo raspi-config
+    ```
+1. Use your keyboard to select “Interface Options”, and press return.
+    ![raspi config](assets/enablePeriph.png)
+1. [Enable the relevant protocols](https://docs.viam.com/installation/prepare/rpi-setup/#enable-communication-protocols) to support our hardware. Since you are using a sensor that communicates over the serial port, enable **Serial Port**.
+  ![enable serial](assets/enableSerial.png)
+1. Confirm the options to enable the serial login shell and serial interface. And reboot the Pi when you're finished.
+
+<!-- ------------------------ -->
+## Configure your machine and peripherals
+Duration: 5
+
+### Configure your machine
+
+1. In [the Viam app](https://app.viam.com/robots), create a machine by typing in a name and clicking **Add machine**. 
+  ![add machine](assets/addMachine.png)
+1. Click **View setup instructions**.
+  ![setup instructions](assets/awaitSetup.png)
+1. Install `viam-server` on the Raspberry Pi device that you want to use to communicate with and control your air sensor. Select the `Linux / Aarch64` platform for the Raspberry Pi to control the air sensor, and leave your installation method as `viam-agent`.
+  ![select platform](assets/selectPlatform.png)
+1. Use the `viam-agent` to download and install `viam-server` on your Raspberry Pi. Follow the instructions to run the command provided in the setup instructions from the SSH prompt of your Raspberry Pi.
```

Comment:
Should we be linking out the docs page on viam-agent? Not sure if it's confusing to say we're installing viam-agent in order to get viam-server running on the device.

Reply:
Added link - definitely confusing. But both are mentioned on that setup page for now.